XamarinCommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -51,7 +51,7 @@
   </PropertyGroup>
   <ItemGroup Condition=" '$(Configuration)'=='Debug' ">
     <!-- Manage Xamarin.Forms version for development -->
-    <PackageReference Include="Xamarin.Forms" Version="4.5.0.356" />
+    <PackageReference Include="Xamarin.Forms" Version="4.5.0.396" />
```

Comment:
Do we actually need to upgrade this? Why?

Reply:
Not sure why that one was changed...I'll revert it